src/utils.js

```diff
@@ -1,37 +1,51 @@
 const BigNumber = require('bignumber.js');
+const RollingMath = require('rolling-math');
 const { ethers } = require('forta-agent');
 
-// create a fake function name
-function getRandomCharacterString(numCharacters) {
-  let result = '';
-  let charCode;
-  for (let i = 0; i < numCharacters; i++) {
-    charCode = Math.floor(Math.random() * 52);
-    if (charCode < 26) {
-      charCode += 65;
-    } else {
-      charCode += 97 - 26;
-    }
-    result += String.fromCharCode(charCode);
-  }
-  return result;
+function getAbi(agentName, abiFile) {
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const abiPath = `../abi/${agentName}`;
+  console.log(`getting: ${abiPath}/${abiFile}`);
+  const { abi } = require(`${abiPath}/${abiFile}`);
+  return abi;
 }
 
-function isAddress(valueString) {
-  return ethers.utils.isHexString(valueString, 20);
+function getInternalAbi(agentType, abiFile) {
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const abiPath = `./${agentType}/internal_abi`;
```

Comment:
Convention-wise, we have tried to stick to directories with hyphens instead of underscores.
```suggestion
  const abiPath = `./${agentType}/internal-abi`;
```